components/server/src/main/java/com/hotels/styx/server/netty/NettyServerBuilder.java

```diff
@@ -51,7 +50,7 @@ public static NettyServerBuilder newBuilder() {
     }
 
     String host() {
-        return firstNonNull(host, "localhost");
+        return host != null ? host : "localhost";
```

Comment:
You could apply @Noddy76 's `Optional.orElse()` technique here.

Reply:
True, and also in AdminServerConfig - I think these are the last ones.